zip-draft-XXXX.rst

```diff
@@ -0,0 +1,269 @@
+::
+
+  ZIP: Unassigned
+  Title: Lockbox for Decentralized Grants Allocation
+  Owners: Skylar Saveland <skylar@free2z.com>
+          Jason McGee <>
+  Credits:
+  Status: Draft
+  Category: Consensus
+  Created: 2024-06-26
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/858>
+
+
+Terminology
+===========
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
+document are to be interpreted as described in BCP 14 [#BCP14]_ when, and
+only when, they appear in all capitals.
+
+.. {Avoid duplicating definitions from other ZIPs. Instead use wording like this:}
+
+.. The terms "Mainnet" and "Testnet" in this document are to be interpreted as
+.. defined in the Zcash protocol specification [#protocol-networks]_.
+
+.. The term "full validator" in this document is to be interpreted as defined in
+.. the Zcash protocol specification [#protocol-blockchain]_.
+
+.. The terms below are to be interpreted as follows:
+
+.. {Term to be defined}
+..   {Definition.}
+.. {Another term}
+..   {Definition.}
+
+
+.. |percentage| replace:: 50%
+
+
+
+Abstract
+========
+
+This ZIP proposes the allocation of |percentage| of the Zcash block reward,
+post-November 2024 halving, to a decentralized "lockbox." Currently, 80% of the
+block reward goes to miners, while 20% is distributed among the Major Grants
+Fund, Electric Coin Company (ECC), and the Zcash Foundation (ZF). If no changes
+are made, this 20% dev fund will expire, resulting in the entire block reward
+going to miners, leaving no block-reward funds for essential protocol
+development, security, marketing, or legal expenses.
+
+The proposed lockbox addresses significant issues observed with [#zip-1014]_,
+such as regulatory risks, inefficiencies in funding organizations instead of
+projects, tax compliance inefficiencies, and centralization. By disbursing
```

Comment:
A future protocol for disbursing these funds should take these issues into account, but
since the current ZIP doesn't propose the disbursement mechanism, we don't know yet
whether it will actually satisfy those regulatory concerns. You mention future disbursement in the motivation section, but the TBD nature of that should be noted in the abstract.